scripts/cli/reportBettererStats.ts

```diff
@@ -0,0 +1,21 @@
+import { betterer } from '@betterer/betterer';
+import { camelCase } from 'change-case';
```

Comment:
Is there any reason we aren't using lodash `camelCase` here to save on adding more dependencies to the project?

Reply:
Good shout!!!!